### PR TITLE
PMM-10444 Add PMM 2.29.1 (#81)

### PR DESCRIPTION
**Pull request by BupycHuk**
it's a cherry-pick of my previous PR to development environment (#81)

### CODE REVIEW

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| api-tests | passed |

**Comment by beatahandzelova** [comment]
@BupycHuk can you pls add pxcOperator 1.11.0 as recommended here? We already have this in check-dev.percona.com/versions/v1/pmm-server/2.29.0.

**Comment by beatahandzelova** [comment]
@BupycHuk can you pls add pxcOperator 1.11.0 as recommended here? We already have this in check-dev.percona.com/versions/v1/pmm-server/2.29.0.

**Comment by BupycHuk** [comment]
@beatahandzelova done

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| api-tests | passed |

File: sources/pmm.2.29.1.pmm-server.json [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+          "1.9.0": {
+            "image_path": "percona/percona-xtradb-cluster-operator:1.9.0",
+            "image_hash": "3783173df8b833028184de686cc98890179a249c7ed5cde8391cdfd751cf990d",
+            "status": "available",
+            "critical": false
+          },
+          "1.10.0": {
+            "image_path": "percona/percona-xtradb-cluster-operator:1.10.0",
+            "image_hash": "73d2266258b700a691db6196f4b5c830845d34d57bdef5be5ffbd45e88407309",
+            "status": "recommended",
```
Comment (beatahandzelova) [inline sources/pmm.2.29.1.pmm-server.json:17]: Not sure if two recommended could cause problems... Change to available?

Comment (BupycHuk) [reply sources/pmm.2.29.1.pmm-server.json]: good point, fixed

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| api-tests | passed |

**Review by beatahandzelova — APPROVED** [review]

**Review by tplavcic — APPROVED** [review]

**Pull request merged**